src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/communitypage/SalesPitchDialog.java

```diff
@@ -11,27 +13,57 @@
   private By salesPitchDialog = By.cssSelector("#CommunityPageBenefitsModal");
 
   @FindBy(css = "#CommunityPageBenefitsModal .community-page-entry-point-button")
-  private WebElement entryPointButton;
+  private WebElement helpOutButton;
 
-  public boolean isVisible() {
+  @FindBy(css = ".community-page-benefits-image")
+  private WebElement image;
+
+  @FindBy(css = ".community-page-benefits-content")
+  private WebElement dialogContent;
+
+  public boolean isDialogVisible() {
     wait.forElementVisible(salesPitchDialog);
 
     return true;
   }
 
-  public boolean isNotVisible() {
+  public boolean isDialogNotVisible() {
     wait.forElementNotVisible(salesPitchDialog);
 
     return true;
   }
 
-  public boolean clickEntryPointButton() {
-    isVisible();
+  public SpecialCommunity clickHelpOutButton() {
+    try {
+      wait.forElementVisible(helpOutButton);
+      helpOutButton.click();
+    } catch (NoSuchElementException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)